pages/features.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: default
+title: Features
+permalink: /features/
+nav_order: 1.5
+---
+
+# Features
+
+This page provides a list of features supported by the application, along with a comparison of feature supported by the QT application.
+
+**This is a complete work-in-progress.**
+
+### General
+
+| Feature                                                    | App     | QT      |
+| ---------------------------------------------------------- | ------- | ------- |
+| Guided setup experience                                    | ✓       | ✗       |
+
+### Wallet management
+
+| Feature                                                    | App     | QT      |
+| ---------------------------------------------------------- | ------- | ------- |
+| Create single-key wallets                                  | ✓       | ?       |
+| Wallet file backup                                         | ✓       | ?       |
+| Wallet file import                                         | ✓       | ?       |
+| Password protection                                        | ✓       | ✓       |
+
+### [Sending]({{ '/1-7-send/' | relative_url }})
+
+| Feature                                                    | App     | QT      |
+| ---------------------------------------------------------- | ------- | ------- |
+| Sending bitcoin                                            | ✓       | ?       |
+| Amount input and "Send all"                                | ✓       | ?       |
+| Address input                                              | ✓       | ?       |
+| Priority-based fee options                                 | ✓       | ?       |
+| Manual fee rate selection                                  | ✓       | ?       |
+| Multiple recipients                                        | ✓       | ?       |
+| Coin selection                                             | ✓       | ?       |
+| Contacts                                                   | ✓       | ?       |
```

Comment:
I don't think all the checked features will be available when the MVP is released. For example the `contacts` and `timelocks` etc.

Something to keep in mind

Reply:
Discussion is good. It would be fantastic if we can have some type of status in there and associated milestones. 

Since we're pre-any-sort-of-launch, maybe this is more of a list of "planned features"? And maybe later we introduce a distinction between "live" and "planned" features. "Live" would be what's merged and "planned" includes anything from early discussion to almost implemented?

I am a bit shying away from committing to having very precise statuses in there, because so far we have not really demonstrated as a group that we want to have very exact specs, docs, project boards, etc. What do you think?